eeraster.py

```diff
@@ -283,17 +290,42 @@ def dem():
       'projects/sat-io/open-datasets/GLO-30').select("b1"))
   return _dem
 
+def _download_raster(raster_name: str):
```

Comment:
Do we need to do any cropping here or specify the projection? If not, where are we doing that?